.github/workflows/flutter_android.yaml

```diff
@@ -0,0 +1,57 @@
+name: Flutter Android
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        description: The working directory relative to the repo root
+        required: true
+        type: string
+      package-name:
+        description: The working directory relative to the repo root
+        required: true
+        type: string
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      # Allows other matrix items to run if one fails
+      fail-fast: false
+      matrix:
+        sdk:
+          - stable
+          - beta
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # 3.1.0
+
+      - name: Git Submodules
+        run: git submodule update --init
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@dbf1fa04f4d2e52c33185153d06cdb5443aa189d # 2.8.0
+        with:
+          cache: true
+          channel: ${{ matrix.sdk }}
+          flutter-version: ${{ matrix.flutter-version }}
+
+      - name: Setup aft
+        run: flutter pub global activate -spath packages/aft
+
+      - name: Update constraints
+        if: matrix.sdk == 'beta'
+        shell: bash
+        run: aft --config=aft.beta.yaml constraints apply
+
+      - name: Bootstrap
+        id: bootstrap
+        timeout-minutes: 10
+        run: aft bootstrap
+
+      - name: Run Android unit tests
+        run: |
+          flutter build apk --debug --verbose
+          ./gradlew :"${{ inputs.package-name}}":testDebugUnitTest
+        working-directory: ${{ inputs.working-directory }}/example/android
```

Comment:
Would almost rather pass the full path in since it's not being used anywhere else